docs/README.md

```diff
@@ -31,34 +31,8 @@ Feathr automatically computes your feature values and joins them to your trainin
 
 ## Running Feathr on Azure with few Simple Steps
 
-1. To enable authentication on the Feathr UI (which gets created as part of the deployment script) we need to create an Azure Active Directory (AAD) application. Currently it is not possible to create one through ARM template but you can easily create one by running the following CLI commands in the [Cloud Shell](https://shell.azure.com/bash)
+Follow the [feathr deployment guide ](./how-to-guides/azure-deployment.md) to run feathr on Azure.
```

Comment:
A suggested way to link to documents are pointing to document site: 
`[feathr deployment guide](https://linkedin.github.io/feathr/how-to-guides/azure-deployment.html)`
[feathr deployment guide](https://linkedin.github.io/feathr/how-to-guides/azure-deployment.html)

Reply:
Interesting, I didn't know that, is there a preference of one over the other?

Reply:
Since all the docs now are in the docs/ folder, the doc builder will automatically convert those relative paths to absolute ones (so the ones that end users see will be `https://linkedin.github.io/feathr/how-to-guides/azure-deployment.html`